src/hotspot/share/prims/universalNativeInvoker.cpp

```diff
@@ -40,38 +40,28 @@ void ProgrammableInvoker::invoke_native(ProgrammableStub stub, address buff, Jav
   assert(thread->thread_state() == _thread_in_vm, "thread state is: %d", thread->thread_state());
 }
 
-JVM_ENTRY(void, PI_invokeNative(JNIEnv* env, jclass _unused, jlong adapter_stub, jlong buff)) {
+JVM_ENTRY(void, PI_invokeNative(JNIEnv* env, jclass _unused, jlong adapter_stub, jlong buff))
```

Comment:
Same here.  There are JNI entry points that should be called coming from native.